API.md

```diff
@@ -249,6 +249,29 @@ request.app['hapi-govuk-question-page'] = {
 }
 ```
 
+Sometimes it may be required to pass additional data to the view. 
+This can be provided via an optional $VIEW$ object, which is passed
+directly to the view:
```

Comment:
Just a minor nit-pick, I think this should say that the _members_ of this of this object are included in the object passed into the view, so will be available for use in the view template.